site/documentation/content/admin-guide/mqtt-adapter-config.md

```diff
@@ -40,6 +40,9 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_MQTT_PORT`<br>`--hono.mqtt.port` | no | `8883` | The secure port that the protocol adapter should listen on.<br>See [Port Configuration]({{< relref "#port-configuration" >}}) below for details. |
 | `HONO_MQTT_SECURE_PROTOCOLS`<br>`--hono.mqtt.secureProtocols` | no | `TLSv1.2` | A (comma separated) list of secure protocols that are supported when negotiating TLS sessions. Please refer to the [vert.x documentation](https://vertx.io/docs/vertx-core/java/#ssl) for a list of supported protocol names. |
 | `HONO_MQTT_TENANT_IDLE_TIMEOUT`<br>`--hono.mqtt.tenantIdleTimeout` | no | `0ms` | The duration after which the protocol adapter removes local state of the tenant (e.g. open AMQP links) with an amount and a unit, e.g. `2h` for 2 hours. See the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration) for an explanation of the format. The value `0ms` disables the timeout. |
+| `hono.mqtt.mapperEndpoints.<mapperName>.host`<br>`--hono.mqtt.mapperEndpoints.<mapperName>.host` | no | `mapper-hono.svc` | The host to which the adapter should connect to call the custom mapper. Replace `<mapperName>` with the name of the mapper to configure. |
```

Comment:
Environment variable names are upper case and use an underscore instead of `.`

Reply:
Fixed.